packages/hash/external-services/kratos/kratos.dev.yml

```diff
@@ -37,6 +37,19 @@ selfservice:
       after:
         password:
           hooks:
+            # TODO - figure out whether this blocks writing to the datastore
+            - hook: web_hook
+              config:
+                url: http://host.docker.internal:5001/kratos-after-registration
+                method: POST
+                body: file:///etc/config/kratos/hooks/after.registration.jsonnet
+                auth:
+                  type: api_key
+                  config:
+                    name: KRATOS_API_KEY
+                    # TODO - figure out how to get this from an environment variable
+                    value: secret
+                    in: header
```

Comment:
Adding this as an environment variable is unfortunately not straightforward, and [would likely involve passing a serialised JSON array](https://github.com/ory/kratos/issues/1792#issuecomment-929096690).

Because this is the dev configuration, I've just copied over the value for the environment variable for now. The production configuration will need a different solution however.